layer-1-contracts/README.md

```diff
@@ -0,0 +1,19 @@
+# Layer 1 Contracts
+
+## Design Goals
+We want our ORU contracts to be built in compliance with the following goals:
+- A **pluggable state machine** which:
+    - allows separation of concerns and parallelizable work
+    - is fully stateless, i.e. all state transitions can be expressed in the form `bytes32 _inputHash, bytes _witness => bytes32 _outputhash` and are only dependent on the immediately preceding pre-state
+- A cryptoeconomic **State Oracle** which:
+    - Provides a cryptoeconomic mechanism to propose, assert, challenge, and finalize optimistic states of the L2 machine
+        - Does NOT require that the entire execution trace is provable, just the execution output
+            - i.e. an honest party may not necessarily ever post a hash of the full trace
+        - DOES require that there is a winning strategy for all, and only for all, possible state proposals
```

Comment:
Should this say something like this?
```suggestion
        - DOES require that there is a winning strategy for correct state proposals
```

I'm not sure what guarantees you can make in case BOTH parties are making incorrect state proposals. But I don't know if you care about that case?

Reply:
Used your suggested wording, and added a bullet below it with the (probably obvious but hey) thing I was trying to convey with "only for all"